app/components/floating_text_input_label/floating_text_input.test.tsx

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {act, fireEvent, render, waitFor} from '@testing-library/react-native';
+import React from 'react';
+import {View} from 'react-native';
+
+import {getDefaultThemeByAppearance} from '@app/context/theme';
+
+import {getStyleSheet} from './styles';
+import {getLabelPositions} from './utils';
+
+import FloatingTextInput, {FloatingTextInputProps, FloatingTextInputRef} from './index';
+
+type ConfigurableProps = Pick<FloatingTextInputProps, 'startAdornment'| 'endAdornment'|'editable' | 'error'|'errorIcon' | 'isKeyboardInput'>
+
+const makeFloatingTextInput = ({endAdornment = null,
+    startAdornment = null,
+    editable = true,
+    error = '',
+    errorIcon = '',
+    isKeyboardInput = true,
+}: ConfigurableProps = {}) => {
+    const theme = getDefaultThemeByAppearance();
+
+    const onPressHandler = jest.fn();
+    const onChangeHandler = jest.fn();
+    const placeholder = 'Placeholder';
+    const label = 'input label';
+    const labelTextStyle = {
+        marginTop: 15,
+    };
+    const value = 'initial value';
+    const testID = 'text_input';
+
+    const ref = React.createRef<FloatingTextInputRef>();
+
+    return {
+        placeholder,
+        onChangeHandler,
+        label,
+        theme,
+        labelTextStyle,
+        value,
+        testID,
+        onPressHandler,
+        ref,
+        renderedElement: render(<FloatingTextInput
+            startAdornment={startAdornment}
+            endAdornment={endAdornment}
+            editable={editable}
+            placeholder={placeholder}
+            onChange={onChangeHandler}
+            label={label}
+            labelTextStyle={labelTextStyle}
+            theme={theme}
+            value={value}
+            error={error}
+            errorIcon={errorIcon}
+            testID={testID}
+            isKeyboardInput={isKeyboardInput}
+            onPress={onPressHandler}
+            ref={ref}
+        // eslint-disable-next-line react/jsx-closing-bracket-location
```

Comment:
Can you add a few line breaks just to avoid the eslint disable? Probably is enough with one line break on line 48 before the `<` and one on line 65 after the `>`.

Reply:
Disabling the rule and changing the indent triggers a lint error related to indentation
**64:1  error  Expected indentation of 8 spaces but found 32  indent**